doc/source/Container.md

```diff
@@ -0,0 +1,132 @@
+# Running watson-conversation-slots-intro in a Container on Bluemix with Kubernetes
+
+This directory allows you to deploy the `watson-conversation-slots-intro` application into a container running on Bluemix, using Kubernetes.
+
+The commands below use environment variables in order to define the specific details of the deployment. Either run the following to export the ENV variables, or substitute your names in the commands or exports:
+```
+$ export CLUSTER_NAME=Watson
```

Comment:
are these example values useful as-is or should this be done w/ placeholders like our usual env.sample?

Reply:
My thinking is we'll eventually move most of this file to an outside repo, which will be the template for this and other redundant stuff.
But each journey will have to contain some specific data, and I believe they should look like the "export ......" lines here. So these lines wouldn't be examples, they are the specifics.